src/TraceEvent/BPerf/BPerfEventSource.cs

```diff
@@ -0,0 +1,312 @@
+﻿//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// This file is best viewed using outline mode (Ctrl-M Ctrl-O)
+//
+// This program uses code hyperlinks available as part of the HyperAddin Visual Studio plug-in.
+// It is available from http://www.codeplex.com/hyperAddin 
+// 
+
+using System;
```

Comment:
Can you please add a test case for BTL files to make sure that subsequent TraceEvent/PerfView changes don't break this support?  You can follow the pattern that is used for ETL and EventPipe traces in https://github.com/Microsoft/perfview/tree/master/src/TraceEvent/TraceEvent.Tests.

Reply:
Yes I'll add a small BTL file (~4MB)

Reply:
Sounds great.